README.md

```diff
@@ -21,11 +21,11 @@ and [J. R. Johansson](https://github.com/jrjohansson)
 [![PyPi Downloads](https://img.shields.io/pypi/dm/qutip?label=downloads%20%7C%20pip&logo=PyPI)](https://pypi.org/project/qutip)
 [![Conda-Forge Downloads](https://img.shields.io/conda/dn/conda-forge/qutip?label=downloads%20%7C%20conda&logo=Conda-Forge)](https://anaconda.org/conda-forge/qutip)
 
-QuTiP is open-source software for simulating the dynamics of closed and open quantum systems.
-The QuTiP library uses the excellent Numpy, Scipy, and Cython packages as numerical backend, and graphical output is provided by Matplotlib.
+QuTiP is an open-source software for simulating the dynamics of closed and open quantum systems.
+The QuTiP library uses Numpy, Scipy, and Cython packages as numerical backends, and Matplotlib for graphical output.
 QuTiP aims to provide user-friendly and efficient numerical simulations of a wide variety of quantum mechanical problems, including those with Hamiltonians and/or collapse operators with arbitrary time-dependence, commonly found in a wide range of physics applications.
 QuTiP is freely available for use and/or modification, and it can be used on all Unix-based platforms and on Windows.
-Being free of any licensing fees, QuTiP is ideal for exploring quantum mechanics in research as well as in the classroom.
+QuTiP does not charge any licensing fees and hence is ideal for exploring quantum mechanics in research as well as in the classroom.
```

Comment:
```suggestion
Being free of any licensing fees, QuTiP is ideal for exploring quantum mechanics in research as well as in the classroom.
```
I think the new phrasing is a slight degradation, so I'd like to keep the original.